### PR TITLE
Document that we do not support backporting migrations

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #7657

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Review by ipanova — APPROVED** [review]

File: docs/bugs-features.rst [hunk 1]
```diff
@@ -89,6 +89,9 @@ For **Backport Requests**
 Only bug fixes from newer versions can be requested to be backported. No features can be backported
 to older versions.
 
+Django migrations are order dependent so we cannot backport any change that has database migrations.
+And thus we also do not ship new migrations in z-stream releases.
```
Comment (bmbouter) [inline docs/bugs-features.rst:93]: nitpick, wdyt?

```suggestion
Thus we do not ship migrations in z-stream releases.
```

Comment (daviddavis) [reply docs/bugs-features.rst:93]: Your suggestion looks good to me except for dropping the word 'new' here. We do ship migrations in z-stream releases but just not new ones. So for users, if they upgrade from say Pulp 3.9 to 3.10.2, they'll still get the migrations from 3.10.0.

Comment (daviddavis) [reply docs/bugs-features.rst:93]: Perhaps this should be, "Thus z-stream releases will not add new migrations."

Comment (bmbouter) [reply docs/bugs-features.rst:93]: +1 to your wording. Agreed.

**Review by bmbouter — APPROVED** [review]
Thank you!

**Pull request merged**